src/verifications/verification.service.ts

```
@@ -0,0 +1,175 @@
+              },
+            },
+          });
+          continue;
+        }
+
+        try {
+          const fileContent = await fs.readFile(file.filePath, 'utf-8');
+          const parsedData = JSON.parse(fileContent);
+
+          const response = await lastValueFrom(
+            this.httpService.post(apiUrl, { credential: parsedData })
```

ESLint

[error] Insert `,`

(prettier/prettier)